core/CHANGELOG.T3T1.md

```
@@ -1335,10 +1355,19 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 [#6349]: https://github.com/trezor/trezor-firmware/pull/6349
 [#6358]: https://github.com/trezor/trezor-firmware/pull/6358
 [#6394]: https://github.com/trezor/trezor-firmware/pull/6394
+[#6435]: https://github.com/trezor/trezor-firmware/pull/6435
 [#6448]: https://github.com/trezor/trezor-firmware/pull/6448
 [#6483]: https://github.com/trezor/trezor-firmware/pull/6483
 [#6501]: https://github.com/trezor/trezor-firmware/pull/6501
```

markdownlint-cli2 (0.22.1)

[warning] Link and image reference definitions should be needed
Unused link or image reference definition: "`#6501`"

(MD053, link-image-reference-definitions)